pym/bob/cmds/build.py

```diff
@@ -1365,13 +1365,23 @@ def doProject(argv, bobRoot):
         help="Do not build (bob dev) before generate project Files. RunTargets may not work")
     parser.add_argument('-b', dest="execute_buildonly", default=False, action='store_true',
         help="Do build only (bob dev -b) before generate project Files. No checkout")
+    parser.add_argument('--build', default=False, action='store_true',
+        help="Do path calculations as if 'bob build', not as 'bob dev'")
     group = parser.add_mutually_exclusive_group()
     group.add_argument('--sandbox', action='store_true', default=False,
         help="Enable sandboxing")
     group.add_argument('--no-sandbox', action='store_false', dest='sandbox',
         help="Disable sandboxing")
     args = parser.parse_args(argv)
 
+    if not '--' in args.args:
+        inputPackages = args.args
+        generatorArguments = []
```

Comment:
Isn't that completely breaking existing workspaces? The generated "buildme" files will not have the double-dash and all arguments will be interpreted as packages.